.github/workflows/pr-review.yml

```
@@ -0,0 +1,347 @@
+      has-frontend-changes: ${{ steps.changes.outputs.frontend }}
+      has-backend-changes: ${{ steps.changes.outputs.backend }}
+      has-docs-changes: ${{ steps.changes.outputs.docs }}
+      
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Detect changes
+        uses: dorny/paths-filter@v2
```

actionlint (1.7.7)

the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)